packages/components/src/internal/components/domainproperties/samples/SampleTypePropertiesPanel.tsx

```diff
@@ -107,6 +108,7 @@ class SampleTypePropertiesPanelImpl extends React.PureComponent<
         nameExpressionInfoUrl: getHelpLink('sampleIDs'),
         nameExpressionPlaceholder: 'Enter a naming pattern (e.g., S-${now:date}-${dailySampleCount})',
         appPropertiesOnly: false,
+        showLinkToStudy: true,
```

Comment:
In the `SampleTypeDesignerImpl` component, this property defaults to `false` and here it defaults to `true`. I would have expected those default values to be the same. any specific reason they are not?

Reply:
Defaults were chosen to correspond to the default behavior according to the previous logic, which was based on the `appPropertiesOnly` property with opposite default values.